pkg/controller/datavolume/import-controller.go

```diff
@@ -101,12 +101,29 @@ func NewImportController(
 	return datavolumeController, nil
 }
 
+// TODO find a better place for this
```

Comment:
There's https://github.com/kubevirt/containerized-data-importer/blob/11fc97e4a64bda2d67a14b3cd26f51dbaf06c3dc/pkg/controller/datavolume/smart-clone-controller.go#L362
But I might be missing a reason that it's not suitable for us

Reply:
problem is that pv claimRef is not a `metav1.Object` so have to sonstruct from namespace/name

Reply:
I would put a common func in `pkg/controller/common` and get rid of the following:
```
datasource-controller.go:        getKey := func(namespace, name string) string {
datavolume/clone-controller-base.go:     getKey := func(namespace, name string) string {
datavolume/pvc-clone-controller.go:      getKey := func(namespace, name string) string {
```